contrib/docker/Dockerfile-alpine

```diff
@@ -1,18 +1,19 @@
 # -*-Dockerfile-*-
 FROM golang:alpine
 RUN apk add alpine-sdk bash llvm5 gnupg xz jq curl-dev sqlite-dev binutils-gold
-ARG LDC_VERSION=1.13.0
+ARG LDC_VERSION=1.16.0
 ENV LDC_VERSION=${LDC_VERSION}
-RUN curl -fsSL "https://github.com/ldc-developers/ldc/releases/download/v${LDC_VERSION}/ldc2-${LDC_VERSION}-alpine-linux-x86_64.tar.xz" |\
+RUN curl -fsSL "https://github.com/ldc-developers/ldc/releases/download/v${LDC_VERSION}/ldc2-${LDC_VERSION}-linux-x86_64.tar.xz" |\
     tar xJf - -C / && \
-    mv "/ldc2-${LDC_VERSION}-alpine-linux-x86_64" "/ldc"
+    mv "/ldc2-${LDC_VERSION}-linux-x86_64" "/ldc"
```

Comment:
You cannot use the general linux version on Alpine, since Alpine requires musl c library, and thus normally binaries don't work.

Reply:
@norbusan 
Unfortunately there is no Alpine Linux version of LDC > 1.13.0 ... and the v1.13.0 fails with:
```
Error relocating /ldc/bin/ldc2: LLVMInitializeMipsTargetInfo: symbol not found
Error relocating /ldc/bin/ldc2: LLVMInitializeMipsTarget: symbol not found
Error relocating /ldc/bin/ldc2: LLVMInitializeMipsTargetMC: symbol not found
Error relocating /ldc/bin/ldc2: LLVMInitializeMipsAsmPrinter: symbol not found
Error relocating /ldc/bin/ldc2: LLVMInitializeMipsAsmParser: symbol not found
```

Reply:
Then we need to drop Alpine. Or find packages distributed by Alpine itself so that they are MUSL based.

Reply:
Fair enough .. will have to see what @jkt628 comes back with, otherwise will have to drop this from Docker